ogr/ogrfeature.cpp

```diff
@@ -562,6 +563,34 @@ OGRGeometryH OGR_F_StealGeometry( OGRFeatureH hFeat )
         OGRFeature::FromHandle(hFeat)->StealGeometry());
 }
 
+/************************************************************************/
+/*                        OGR_F_StealGeometryEx()                         */
+/************************************************************************/
+
+/**
+ * \brief Take away ownership of geometry.
+ *
+ * Fetch the geometry from this feature, and clear the reference to the
+ * geometry on the feature.  This is a mechanism for the application to
+ * take over ownership of the geometry from the feature without copying.
+ * Sort of an inverse to OGR_FSetGeometryDirectly().
```

Comment:
Instead of "sort of an inverse" (which sounds too informal) I'd suggest something like "This is the functional opposite of OGR_F_SetGeomFieldDirectly" or "This method performs the inverse operation to OGR_F_SetGeomFieldDirectly".

Reply:
Done in commit 1601d02